zokrates_core_test/tests/tests/array_if.json

```diff
@@ -1,5 +1,6 @@
 {
 	"entry_point": "./tests/tests/array_if.zok",
+	"curve": "Bn128",
```

Comment:
Just abusing this file for a more general remark on test cases:
We should have integration tests for all curves supported.

Reply:
Now accepting a list here